apps/mull-ui/src/app/components/contact-row/contact-row.scss

```diff
@@ -0,0 +1,49 @@
+@import '../../../variables.scss';
+
+.contact-container {
+  display: flex;
+  margin-top: 2rem;
+  justify-content: space-between;
+  .user-information {
+    display: flex;
+    flex-direction: row;
+  }
+  .user-profile-picture {
+    width: 3.5rem;
+    height: 3.5rem;
+    object-fit: cover;
+    border-radius: 50%;
+    border: 1px solid black;
+  }
+  .user-details {
+    display: flex;
+    padding: 0.3rem;
+    flex-direction: column;
+    .userName {
```

Comment:
💡 Careful with the naming. So far we've been using the kebab casing (your-name-here) for scss classes, and here you're mixing both kebab and camel cases.